certified-connectors/Alkymi/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1301 @@
+            }
+          }
+        }
+      }
+    },
+    "/api/v2/executions/{uuid}/": {
+      "get": {
+        "description": "Retrieve a specific execution instance.\\nhttps://alkymi-api.readme.io/reference#get-execution-list",
+        "summary": "Get Execution Information",
+        "tags": [
+          "executions"
+        ],
+        "operationId": "executions_retrieve",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "uuid",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "description": "Execution UUID",
+            "x-ms-summary": "Execution UUID",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/Execution"
+            }
+          }
+        }
+      }
+    },
+    "/api/v2/pipelines/": {
+      "get": {
+        "description": "List all patterns on tenant",
+        "summary": "List Patterns",
+        "tags": [
+          "pipelines"
+        ],
+        "operationId": "pipelines_list",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "page",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "description": "A page number within the paginated result set.",
+            "x-ms-summary": "A page number within the paginated result set.",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/PaginatedPipelineConfigurationList"
+            }
+          }
+        }
+      }
+    },
+    "/api/v2/pipelines/{name}/": {
+      "get": {
+        "description": "Retrieve a specific pattern instance. The returned pattern record may require additional parsing before using downstream in the workflow.",
+        "summary": "Get Pattern Information",
+        "tags": [
+          "pipelines"
+        ],
+        "operationId": "pipelines_retrieve",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Pattern Name",
+            "x-ms-summary": "Pattern Name",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/PipelineConfiguration"
+            },
+            "headers": {}
```

Comment:
@tori-alkymi Empty property values can be removed unless you have a specific reason to keep them. Applies the same comment for all the similar empty properties in the swagger.